tests/buffer/buffer_api_common.h

```diff
@@ -436,12 +436,13 @@ void test_buffer(util::logger &log, cl::sycl::range<dims> &r,
 template <typename T, int numDims>
 void test_type_reinterpret(util::logger &log) {
   static constexpr size_t inputElemsPerDim = 4;
-  std::vector<uint8_t> reinterpretInputData(sizeof(T) * inputElemsPerDim *
-                                            numDims);
+  auto numElems = inputElemsPerDim;
+  for (int i = 1; i < numDims; ++i)
+    numElems *= inputElemsPerDim;
```

Comment:
```suggestion
  for (int i = 1; i < numDims; ++i) {
    numElems *= inputElemsPerDim;
  }
```

We could argue about what is better, but please keep the braces at least for consistency with other code